lib/models/network/AbstractAccount.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:defi_wallet/models/address_book_model.dart';
+import 'package:defi_wallet/models/history_model.dart';
+import 'package:defi_wallet/models/token_model.dart';
+
+abstract class AbstractAccount {
```

Comment:
My propose in this model store hashed password, encrypted privateKey, publicKey, etc. But all address info, balances, tokens, etc move to network model, or something like this. Because address will be changed for different networks